org.seventhstring.Transcribe.yml

```diff
@@ -0,0 +1,40 @@
+app-id: org.seventhstring.Transcribe
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: transcribe
+finish-args:
+#TODO: figure out if its possible to only grant permissions 
+# on ~/.Transcribe!7 and xdg-music:ro, for now we use home
+# as a functional compromise
```

Comment:
please do.

Reply:
The problem i faced here is that when i use `--filesystem=~/.Transcribe!7` or `persist=.Transcribe!7` it does not work as expected. The application will not persist that file in the host then and ask for a license key in the next startup again.

Reply:
Is there a way to only allow access to the file `.Transcribe!7` in home? i could not figure it out i have to admit.